tests/check_framework/test_multi_db.py

```diff
@@ -0,0 +1,53 @@
+from django.db import connections, models
+from django.test import TestCase, mock
+from django.test.utils import override_settings
+
+from .tests import IsolateModelsMixin
+
+
+class TestRouter(object):
+    """
+    Dummy router that would route to the 'other' database
+    if the model name starts with 'Other'.
+    """
+    def db_for_read(self, model, **hints):
```

Comment:
Are all the methods required? My intuition is that `allow_migrate()` would be sufficient for testing.